docs-src/concepts.md

```diff
@@ -173,7 +173,31 @@ handling this:
 
 * TODO
 
-### Extensibility
+### Conformance
+
+As this API aims to cover a wide set of implementations and use cases,
+it will not be possible for all implementations to support all
+features at a given time. However, we do expect that the set of
+features that are supported to be converging. To model this in the
+API, we are taking a similar approach as with
+[SIG-CONFORMANCE][sig-conformance] work on conformance
+profiles. Features as described in the API spec will be divided into
+three major categories:
+
+* **CORE** features are those that will always work in a portable way
+  for ALL implementations that support the API.
```

Comment:
I was using this as a guiding principle to decide which bucket do the features go into and ran into two problems:

1. If we decide a feature is supported in core, and there comes an implementer who says their proxy can't support feature x, how is that resolved? We need to document this for future-selves or other future maintainers of this API.
2. How is "ALL" defined? Quite a few implementers are working together in this working group but we probably do have some that are not represented.

Reply:
Adding some language to address your questions -- please take a look

Reply:
It resolves some but not more about the process of how to define "ALL", which is a larger topic and seems out of out of scope from this PR.